decks/card_definitions.json

```diff
@@ -1451,6 +1451,602 @@
+            }
+        ]
+    },
+    {
+        "card_id": "hSD03-011",
+        "card_type": "holomem_spot",
+        "bloom_blocked": true,
+        "card_names": ["laplus_darkness"],
+        "rarity": "u",
+        "colors": ["none"],
+        "hp": 60,
+        "baton_cost": 1,
+        "tags": [
+            "#JP",
+            "#SecretSocietyholoX",
+            "#Shooter"
+        ],
+        "arts": [
+            {
+                "art_id": "doroboconstructionagriculturalminister",
+                "costs": [{ "color": "any", "amount": 1 }],
+                "power": 10
+            },
+            {
+                "art_id": "illdefinitelyletyoueatit",
+                "costs": [{ "color": "any", "amount": 2 }],
+                "power": 20,
+                "art_effects": [
+                    {
+                        "timing": "before_art",
+                        "effect_type": "draw",
+                        "conditions": [{ "condition": "cards_in_hand", "amount_max": 2 }],
+                        "max_in_hand": 3
+                    }
+                ]
+            }
+        ]
+    },
+    {
+        "card_id": "hSD03-012",
+        "card_type": "support",
+        "sub_type": "event",
+        "card_names": ["dorobo_construction"],
+        "rarity": "u",
+        "limited": true,
+        "colors": [],
+        "play_conditions": [{ "condition": "cards_in_hand", "amount_max": 7 }],
+        "effects": [
+            {
+                "effect_type": "choose_cards",
+                "from": "deck",
+                "look_at": 4,
+                "destination": "hand",
+                "amount_min": 0,
+                "amount_max": 4,
+                "requirement": "holomem_named",
+                "requirement_names": ["nekomata_okayu", "takane_lui", "ookami_mio", "shirakami_fubuki", "laplus_darkness", "inugami_korone"],
+                "reveal_chosen": true,
+                "remaining_cards_action": "order_on_bottom"
+            }
+        ]
+    },
+    {
+        "card_id": "hSD03-013",
+        "card_type": "support",
+        "sub_type": "mascot",
+        "card_names": ["okanyan"],
+        "rarity": "c",
+        "colors": [],
+        "effects": [{ "effect_type": "attach_card_to_holomem" }],
+        "attached_effects": [
+            {
+                "timing": "on_take_damage",
+                "conditions": [{ "condition": "attached_owner_is_location", "condition_location": "center_or_collab" }],
+                "effect_type": "reduce_damage",
+                "amount": 10
+            },
+            {
+                "timing": "before_archive_cheer",
+                "conditions": [
+                    { "condition": "attached_to", "required_member_name": "nekomata_okayu" }
+                ],
+                "effect_type": "choice",
+                "choice": [
+                    {
+                        "full_english_text": "MASCOT_HSD03_013_PARTIAL_TEXT",
```

Comment:
The current text is ``Archive %s less from Hand``. Unless we supply more info in the card definition, it would be hard to construct the string correctly.

Reply:
Maybe we can just change it to Archive %s less?

Reply:
Alternatively we can add some helper info to the effect like "location": "cheer" or "hand" or something